micrometer/src/main/java/feign/micrometer/FeignObservationDocumentation.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2012-2022 The Feign Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package feign.micrometer;
+
+import io.micrometer.common.docs.KeyName;
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationConvention;
+import io.micrometer.observation.docs.ObservationDocumentation;
+
+/**
+ * {@link ObservationDocumentation} for Feign.
+ * 
+ * @since 1.10.0
+ */
+public enum FeignObservationDocumentation implements ObservationDocumentation {
+
+  DEFAULT {
+    @Override
+    public Class<? extends ObservationConvention<? extends Observation.Context>> getDefaultConvention() {
+      return DefaultFeignObservationConvention.class;
+    }
+
+    @Override
+    public KeyName[] getLowCardinalityKeyNames() {
+      return HttpClientTags.values();
+    }
+  };
+
+  enum HttpClientTags implements KeyName {
+
+    STATUS {
+      @Override
+      public String asString() {
+        return "status";
```

Comment:
I've made a mistake in the original PR, let's go with the semantic conventions from OTel https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md

- `method` -> `http.method`
- `status` -> `http.status_code`
- `target.port` -> `net.peer.port`
- `target.host` -> `net.peer.name`
- `uri` -> `http.url`

I don't see `scheme` in the conventions so we can go with `http.scheme`

Reply:
What if you merge my chances into your PR and then you can do the required changes?

Reply:
No problem